UI.py

```diff
@@ -211,7 +211,63 @@ def CreateIsMemberWorkingOptions():
     ]
 
     return member_options
+
+def CreateGetEndOfMonthCalculationsOptions():
+    end_of_month_calculations_options = [
+        create_option(
+            name = "month",
+            description = "number of the desired month (optional) leave empty for current month",
+            option_type = SlashCommandOptionType.INTEGER,
+            required = False
+        ),
+        create_option(
+            name = "year",
+            description = "number of the desired year (optional) leave empty for current year",
+            option_type = SlashCommandOptionType.INTEGER,
+            required = False
+        )
+    ]
+
+    return end_of_month_calculations_options
     
+def CreateGetEndOfMonthCalculationsEmbed(month = None, year = None):
+    month = month or datetime.datetime.now().month
+    year = year or datetime.datetime.now().year
+    embed = discord.Embed(
+        title = f'End of Month Calculations',
```

Comment:
Let's call this a report, not a calculation

Reply:
Change this across the entire pr please

Reply:
Done